deploy/openshift/ui-plugin/configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
       default_type       application/octet-stream;
       keepalive_timeout  65;
       server {
-        listen              9443 ssl;
+        listen              [::]:{{ .PluginPort }} ipv6only=off ssl;
```

Comment:
Do this work for ipv4 single stack ?

Reply:
yes, verified this also with IPv6 single stack and IPv4 single stack

Reply:
@qinqon , i verified it works also on a Dual-Stack IPv4+IPv6 cluster:
```
$ oc get network cluster -o json | jq .status
{
  "clusterNetwork": [
    {
      "cidr": "10.128.0.0/14",
      "hostPrefix": 23
    },
    {
      "cidr": "fd02::/48",
      "hostPrefix": 64
    }
  ],
  "clusterNetworkMTU": 1400,
  "networkType": "OVNKubernetes",
  "serviceNetwork": [
    "172.30.0.0/16",
    "fd03::/112"
  ]
}
```

from the nmstate-console-plugin pod logs:
```
::ffff:10.128.0.137 - - [26/Jul/2023:14:57:50 +0000] "GET /plugin-manifest.json HTTP/1.1" 200 3269 "https://console-openshift-console.apps.<REDACTED>/k8s/ns/openshift-nmstate/pods/nmstate-console-plugin-5d6ff7f8bd-stqf9/logs" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
```